### DISCUSSION TITLE
How to add extra plugins to the crepe editor?

### DISCUSSION THREAD

**Discussion by VincentK-ZA** [Milkdown/milkdown#1432] [Q&A]
I've been trying to add the [collab plugin](https://milkdown.dev/docs/api/plugin-collab) to the [crepe editor](https://milkdown-storybook.vercel.app/?path=/story/crepe-nord--empty) without any luck.  

I run into a similar issue when trying to use the [listener plugin](https://milkdown.dev/docs/api/plugin-listener), so I've created a [stackblitz example](https://stackblitz.com/edit/vitejs-vite-sonqfe?file=src%2Flib%2FCrepe.svelte) showing the error.

The error message I receive in the dev console is `MilkdownError: Timer "InitReady" not found, do you forget to record it?`

For the collab plugin it's just `EditorViewReady` timer instead.

As seen in the `Crepe.svelte` file on stackblitz, I am trying to add the plugin as per usual (see the `Milkdown.svelte` file) directly to the underlying `Editor`, however, it seems the plugins that `listener` depends on either aren't available yet, or have already been destroyed.

So any help or examples showing how to register additional plugins would be appreciated.

**Comment by Saul-Mirone** [comment]
It's a bug caused by wrong config of rollup. Now I've fixed it in 7.5.2 and you can register the plugin as usual.
